### PR TITLE
Oak SDK Implement remaining structs and handles

### PR DESCRIPTION
**Pull request by jul-sh**
Fixes http://b/312897453

### CODE REVIEW

File: oak_dice/src/evidence.rs [hunk 1]
```diff
@@ -114,7 +114,7 @@ impl RootLayerEvidence {
 static_assertions::assert_eq_size!([u8; 2320], RootLayerEvidence);
 
 /// Attestation evidence for non-root layers.
-#[derive(AsBytes, FromZeroes, FromBytes)]
+#[derive(AsBytes, FromZeroes, FromBytes, Clone, Copy)]
```
Comment (jul-sh) [inline oak_dice/src/evidence.rs:117]: Implementing this as applications receiving AttestationEvidence will likely want to clone it (based on current usage)

Comment (conradgrobler) [reply oak_dice/src/evidence.rs:117]: I agree with `Clone`, but not sure `Copy` is needed as it might lead to accidental cloning via assignment.

**Review by conradgrobler — APPROVED** [review]
Thanks.

File: oak_dice/src/evidence.rs (hunk 1 above)
Comment (jul-sh) [reply oak_dice/src/evidence.rs]: oki, removed it

**Pull request merged**